pkg/controller/configmap/config.go

```diff
@@ -431,3 +455,29 @@ func optionInConfigMap(configMap *corev1.ConfigMap, section string, key string)
 	}
 	return false
 }
+
+func getOptionInConfigMap(configMap *corev1.ConfigMap, section string, key string) string {
+	cfg, err := ini.Load([]byte(configMap.Data[operatortypes.ConfigMapDataKey]))
+	if err != nil {
+		log.Error(err, "Failed to load ConfigMap")
+		return ""
+	}
+	sec, err := cfg.GetSection(section)
+	if err != nil {
+		return ""
+	}
+	if sec.HasKey(key) {
+		return sec.Key(key).Value()
+	}
+	return ""
+}
+
+func IsMTUChanged(currConfigMap *corev1.ConfigMap, prevConfigMap *corev1.ConfigMap) bool {
+	currMtu := getOptionInConfigMap(currConfigMap, "nsx_node_agent", "mtu")
+	prevMtu := getOptionInConfigMap(prevConfigMap, "nsx_node_agent", "mtu")
```

Comment:
just to make sure I read the code correctly, these will always be strings right?

Reply:
Yes, values are all strings
https://godoc.org/gopkg.in/go-ini/ini.v1#Key.Value

Reply:
ok, we're good then ;)